package.json

```diff
@@ -8,17 +8,19 @@
     },
     "exports": "./dist/index.js",
     "module": "./dist/index.js",
+    "types": "./dist/index.d.ts",
     "files": [
         "dist"
     ],
     "scripts": {
         "start": "vite demo/ --config vite.config.mts",
-        "build": "npm install; vite build; npm prune --production",
-        "build:only": "vite build",
+        "build": "npm install && tsc --p ./tsconfig-build.json && vite build && npm prune --production",
+        "build:only": "tsc --p ./tsconfig-build.json && vite build",
```

Comment:
vite build cannot integrate ts compiling directly ?

Reply:
Nope, I will change all these scripts in an other PR